test/gui/shared/steps/steps.py

```diff
@@ -378,8 +380,7 @@ def createFile(context, filename, username=None):
     f = open(join(syncPath, filename), "w")
     f.write(fileContent)
     f.close()
-
-
+    
```

Comment:
please revert this change back to 2 empty lines between the two step definitions/functions